gramex/handlers/mlhandler.py

```diff
@@ -351,3 +351,16 @@ def delete(self, *path_args, **path_kwargs):
                 getattr(self, f'_delete_{item}')()
             except AttributeError:
                 raise HTTPError(BAD_REQUEST, f'Cannot delete {item}.')
+
+
+class MLPredictor(FormHandler):
+
+    @classmethod
+    def setup(cls, model, data, **kwargs):
```

Comment:
`data` does not need to be a parameter. We will allow `url:` etc at the top level.

Let's replace `model` with `config_dir` and ask users to place their `.pkl` under `config_dir` as `model.pkl`. Makes it consistent with MLHandler. (Might mean more work for the user, but let's favor consistency for now, and simplify it for users with another config later if required.)